examples/coap-server.c

```diff
@@ -1475,6 +1725,9 @@ main(int argc, char **argv) {
 
   init_resources(ctx);
 
+  /** Define the options to ignore when setting up cache-keys */
```

Comment:
As this comment is not meant for doxygen it should start with `'/* '` only.

Reply:
Corrected.